### ISSUE TITLE
Request: Support for google Meet/hangouts

### ISSUE THREAD

**Issue by a-zndr** [bdkjones/MuzzleApp#21]
meet doesn't appear to have support, I suspect no browser based meet tool is yet

**Comment by paulvantuyl** [comment]
It partially works for me with Google Hangouts Meet, macOS 10.12.6, Chrome 59.0.3071.115.

**Works**: Turns on DND when starting screen share.
**Doesn’t work**: Turning off DND when ending screen share.

**Comment by bdkjones** [comment]
I can't diagnose this without additional information. Please use a camera to record the behavior on your Mac. (Don't record your screen, as Muzzle will fire on the screen-capture.)

**Comment by tylerclark** [comment]
I am seeing the same problem @paulvantuyl

**Comment by tylerclark** [comment]
https://youtu.be/azuS1zzV9bI @bdkjones

**Comment by tylerclark** [comment]
@bdkjones Is this app still in development or no? No response in months.

**Comment by bdkjones** [comment]
Yep! It's just (A) a side project and (B) I can't add support for apps/tools that don't use the modern screensharing API on macOS. That's what Muzzle monitors to determine when screensharing starts. Older tools (and poorly-written ones) use a "take 30 screenshots a second and stitch them together" approach, which spikes CPU and kills battery.

**Comment by tylerclark** [comment]
I would love to contribute! My company (Salesforce) really likes us to be involved in cool things like this. I use it all the time, but I have to remember to turn OFF dnd after a meeting. Would love to fix it so after the Google Hangout/Meet it turns it off automatically.
